README.md

```diff
@@ -416,3 +417,17 @@ community engagement
 * hacktoberfest
 * bounty? - proposal to be included in a bounty programme
 * bounty - included in Status Open Bounty
+
+## Caching requirements
```

Comment:
I'd also move this up above a bit higher so it's not buried as a footnote like the triage labels are